CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_stripe_source v0.11.1
+[PR [#74](https://github.com/fivetran/dbt_stripe_source/pull/74)] includes the following updates:
+
+## 🚨 Breaking Changes 🚨
+- Updates the `unique_invoice_line_item_id` uniqueness test in `stg_stripe__invoice_line_item` to include `invoice_id`. This is because `unique_invoice_line_item_id` (`unique_id` in the raw source `invoice_line_item` table) was part of an older version of Stripe that was included in the new version to help migrate internal references. See the Stripe [API update](https://stripe.com/docs/upgrades#2019-12-03) for more information. The Fivetran connector persists this in order to resolve the pagination break issue for invoice line items that was introduced by the [API update](https://stripe.com/docs/upgrades#2019-12-03).
```

Comment:
Request to update this based on feedback from the test updates.

Reply:
Additionally, is there a reason this is listed as a breaking change? If it is a breaking change, we will need to bump the version to v0.12.0 and also make a breaking change downstream in the dbt_stripe package.

I am not entirely sure a breaking change is needed here.

Reply:
Yes update based on above! And true, I changed this to a bugfix.

Reply:
Reminder to make the appropriate updates here based on the other comment.